pkg/render/logstorage/esgateway/esgateway.go

```diff
@@ -270,17 +270,7 @@ func (e esGateway) esGatewayDeployment() *appsv1.Deployment {
 					Image:        e.esGatewayImage,
 					Env:          envVars,
 					VolumeMounts: volumeMounts,
-					ReadinessProbe: &corev1.Probe{
-						Handler: corev1.Handler{
-							HTTPGet: &corev1.HTTPGetAction{
-								Path:   "/health",
-								Port:   intstr.FromInt(Port),
-								Scheme: corev1.URISchemeHTTPS,
-							},
-						},
-						InitialDelaySeconds: 10,
-						PeriodSeconds:       5,
-					},
```

Comment:
When we add this back in (after ES status and Kibana status are eliminated) we can add a value for `failureThreshold` so that it attempts multiple times before it deems es-gateway unhealthy. 

> failureThreshold: When a probe fails, Kubernetes will try failureThreshold times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready. Defaults to 3. Minimum value is 1.